### DISCUSSION TITLE
get watch battery in home assistant

### DISCUSSION THREAD

**Discussion by starschild86** [house-of-abbey/GarminHomeAssistant#114] [General]
hi,
i would like to install this app as a priority to get battery information (to have automations based on battery level, e.g. remember me to recharge when it's at 25% or stop recharging when it's at 80% to preserve battery longevity)
at the moment i'm not interested in controlling actions from the watch but the future might bring more.
what would be a barebone setup to get this? i went through the readme but i'm not sure.
thanks,
Nicola

**Comment by josephabbey** [comment]
There is currently no way to install this app without the dashboard, but you can give it a blank dashboard, remember that in order for the sensors to update, you must open the app after the watch restarts.
